pkg/controller/clusterautoscaler/clusterautoscaler_test.go

```diff
@@ -36,18 +36,19 @@ const (
 )
 
 var (
-	ScaleDownUnneededTime        = "10s"
-	ScaleDownDelayAfterAdd       = "60s"
-	MaxNodeProvisionTime         = "30m"
-	PodPriorityThreshold   int32 = -10
-	MaxPodGracePeriod      int32 = 60
-	MaxNodesTotal          int32 = 100
-	CoresMin               int32 = 16
-	CoresMax               int32 = 32
-	MemoryMin              int32 = 32
-	MemoryMax              int32 = 64
-	NvidiaGPUMin           int32 = 4
-	NvidiaGPUMax           int32 = 8
+	ScaleDownUnneededTime               = "10s"
+	ScaleDownUtilizationThreshold       = "0.4"
```

Comment:
What is the default value for this in the autoscaler itself? I was under the impression it was 0.5

Reply:
that's correct